.github/workflows/main_l2_proving.yaml

```diff
@@ -0,0 +1,64 @@
+name: L2 (SP1 Backend)
```

Comment:
maybe rename to `main_prover` to be consistent with `pr_prover`?

Reply:
Renamed here [9d89d69](https://github.com/lambdaclass/ethrex/pull/2291/commits/9d89d69b80079440162ac4df9a430313add51ad9)